.github/workflows/ci.yml

```diff
@@ -197,6 +202,12 @@ jobs:
           FAST_LINT_TEST_CASES: 1
         run: vendor/bin/paraunit coverage --testsuite unit --pass-through=--exclude-group=covers-nothing --clover=build/logs/clover.xml
 
+      - name: Run mutation tests (Infection)
+        if: matrix.run-mutation-tests == 'yes'
+        env:
+          FAST_LINT_TEST_CASES: 1
+        run: composer infection
```

Comment:
*minor*

inconsistency: before we run `composer test` (and not `composer PHPUnit`) , but here we run `composer infection` (and not `composer mutation` etc)

Reply:
I'll use `test:mutation` here (also available). `infection` is just a handy alias.

Reply:
In the end I've used totally dedicated script for CI.